trame_vtk/widgets/vtk/common.py

```diff
@@ -1,5 +1,6 @@
 from trame_client.widgets.core import AbstractElement
 from trame_vtk.modules import common
+from trame_vtk.modules.vtk import HAS_VTK_WEB, IMPROPER_VTK_MSG
```

Comment:
That part is not correct as it needs to work for both VTK and ParaView... 

You need to use `MODULE` within the class to check if they are valid.
Then within the MODULE when it is VTK you can raise that exception.

Reply:
Like `MODULE.validate()`